src/Essentials/src/Types/LocationExtensions.android.cs

```diff
@@ -24,7 +24,11 @@ internal static Location ToLocation(this AndroidLocation location) =>
 			{
 				Latitude = location.Latitude,
 				Longitude = location.Longitude,
+#if ANDROID34_0_OR_GREATER
```

Comment:
This probably will not work as you expected. The better is it to use 

`if (OperatingSystem.IsAndroidAtLeast(34))`

Reply:
thanks, will do!